### DISCUSSION TITLE
Can i use vim-polyglot instead of treesitter?

### DISCUSSION THREAD

**Discussion by bty102** [projekt0n/github-nvim-theme#358] [Q&A]

**Comment by tmillr** [comment]
Neovim is heavily integrated-with, and generally works best with, treesitter. Using the [treesitter plugin](https://github.com/nvim-treesitter/nvim-treesitter) is probably the easiest way to install and set it up for the different langs you need. Currently, most of the syntax highlighting provided by this plugin is done so via treesitter, although there is support for legacy highlight groups as well. So it should still work in theory, but you will very likely get more accurate highlighting using treesitter.

Why do you need vim-polyglot?
